pkg/client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx

```diff
@@ -10,7 +10,6 @@ import {
   Tooltip,
 } from "@patternfly/react-core";
 import { Link } from "react-router-dom";
-import { stringify } from "yaml";
```

Comment:
Not relevant to this PR. (I know there are a lot of unused import trailing behind which I've always tried to eliminate though!)

Reply:
Yeah its just a removal of an unused import.